<ui version="4.0" >
 <class>debunk2</class>
 <widget class="QDialog" name="debunk2" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>debuNK2</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QTableWidget" name="parsedTable" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>481</width>
     <height>381</height>
    </rect>
   </property>
   <property name="dragEnabled" >
    <bool>true</bool>
   </property>
   <property name="alternatingRowColors" >
    <bool>true</bool>
   </property>
   <row>
    <property name="text" >
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text" >
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Address</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="nk2Location" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops" >
    <bool>true</bool>
   </property>
   <property name="editable" >
    <bool>true</bool>
   </property>
   <property name="autoCompletion" >
    <bool>true</bool>
   </property>
   <property name="duplicatesEnabled" >
    <bool>false</bool>
   </property>
   <item>
    <property name="text" >
     <string>Trying to find autocompletion (NK2) files...</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="nk2Locator" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>25</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Locate NK2...</string>
   </property>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>500</x>
     <y>60</y>
     <width>83</width>
     <height>197</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Export to file:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioCSV" >
      <property name="toolTip" >
       <string>Comma-separated values</string>
      </property>
      <property name="text" >
       <string>CSV</string>
      </property>
      <property name="checked" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTSV" >
      <property name="toolTip" >
       <string>Tab-separated values</string>
      </property>
      <property name="text" >
       <string>TSV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioSSV" >
      <property name="toolTip" >
       <string>Semicolon-separated values</string>
      </property>
      <property name="text" >
       <string>SSV</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioSyncML" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>SyncML</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioVCard" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>VCard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="export" >
      <property name="text" >
       <string>E&amp;xport...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>450</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="statusBar" >
   <property name="geometry" >
    <rect>
     <x>138</x>
     <y>450</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>23</height>
    </size>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="text" >
    <string>debuNK2 (C) 2006 havard@dahle.no</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>nk2Location</tabstop>
  <tabstop>nk2Locator</tabstop>
  <tabstop>parsedTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
